pkg/dnfjson/dnfjson.go

```diff
@@ -776,7 +783,7 @@ func run(dnfJsonCmd []string, req *Request) ([]byte, error) {
 	cmd := exec.Command(ex, args...)
 	stdin, err := cmd.StdinPipe()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("calling %s failed: %w", ex, err)
 	}
 
 	cmd.Stderr = os.Stderr
```

Comment:
(unrelated to the PR) - this is slightly unusual, generally capturing stderr to provide more meaningful error message programmatically is a good idea.

Reply:
`osbuild-depsolve-dnf` prints some errors to stderr (check the `printe()` calls in the script) but all meaningful (captured) info and errors are returned to stdout.  Piping stderr straight out is done because libdnf will print a lot of useless warnings that we don't care about.  We _could_ parse them and filter out the ones we expect (`No match for group package <whatever>`) and maybe log anything else.`